components/Addressing/interfaces.rst

```diff
@@ -0,0 +1,99 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_addressing_model_address-interface:
+
+AddressInterface
+~~~~~~~~~~~~~~~~
+
+This interface should be implemented by a model representing the customer's address.
+
+.. note::
+   This interface extends :ref:`component_resource_model_timestampable-interface`. |br|
+   For more detailed information go to `Sylius API AddressInterface`_.
+
+.. _Sylius API AddressInterface: http://api.sylius.org/Sylius/Component/Addressing/Model/AddressInterface.html
+
+.. _component_addressing_model_country-interface:
+
+CountryInterface
+~~~~~~~~~~~~~~~~
+
+This interfaces should be implemented by a model representing a country.
```

Comment:
Agree with plural for models, but have to disagree with the bolding as i only use bold for referring to classes already implemented by default or as an emphasis to important words in basic usage.